xbit/routes/controls/user.js

```diff
@@ -0,0 +1,110 @@
+
+var async = require('async');
```

Comment:
looks your are defining routes.

It would be better to use router, so you have
var router = express.router();

router.get(...)
router.post(...)

in parent path, you can then simply write

router.use('/foo', require('foo.js'))